pages/linux/uuid.md

```diff
@@ -0,0 +1,32 @@
+# uuid
+
+> Generate and decode Universally Unique Identifiers (UUID).
+> More information: <https://manned.org/uuid>.
```

Comment:
Just a note, the official man page would be at http://www.ossp.org/man/man.cgi/pkg/lib/uuid/uuid_cli.pod

But it is at 500 error for a few days already, I have just written an email there.

The base page is at http://www.ossp.org/pkg/lib/uuid/

Manned.org displays the same man page as installed locally with the command, so it can probably stay as it is.

Reply:
We can change the link as soon as the website works again, but for now `manned.org` is perfectly fine.